### ISSUE TITLE
Isolated pnpm lockfile is missing overrides

### ISSUE THREAD

**Issue by Nr9** [0x80/isolate-package#86]
Using pnpm, when deploying to firebase we have the following error:
```
Scope: all 11 workspace projects
 ERR_PNPM_LOCKFILE_CONFIG_MISMATCH  Cannot proceed with the frozen installation. The current "overrides" configuration doesn't match the value found in the lockfile

Update your lockfile using "pnpm install --no-frozen-lockfile"; Error ID: be874b7a
Build failed with status: FAILURE and message: installing pnpm dependencies: (error ID: 8622936f):
```

The created `pnpm-lock.yaml` does not contain the `overrides` section from the original `pnpm-lock.yaml`

Note: we are migrating our project from `npm` to `pnpm`. Isolate works correctly when using `npm`

**Comment by 0x80** [comment]
Overrides were taken out explicitly to solve someone else's issue with them. See #55. I don't have experience with them myself, but if they are sometimes necessary and sometimes breaking things, we could make it a configuration option to include/exclude them.

Arent the overrides applied locally to the dependencies? How should they end up in the deployment?

**Comment by 0x80** [comment]
I can reproduce the issue with an override in the root manifest with pnpm v9. Possibly the previous solution only worked for v8 or the lockfile pruning that I added later messed things up...

**Comment by 0x80** [comment]
It appears that the later added prune step is stripping the overrides from the lockfile. I have added them back and published `isolate-package@1.17.0-0` under tag `next`. Please give it a try an see if it works for you.

**Comment by Nr9** [comment]
Tested deploy with new version. It works! Thank you.
